src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -47,6 +49,20 @@ class CrashModule(
         }
     }
 
+    private fun uploadDeobfuscatedCrashes(issue: Issue, crashes: List<Pair<AttachmentUtils.TextDocument, Crash>>) {
+        val minecraftCrashesWithDeobf = crashes
+            .map { it.first.name to it.second }
+            .filter { it.second is Crash.Minecraft }
+            .map { it.first to (it.second as Crash.Minecraft).deobf }
+            .filter { it.second != null }
+            .filterNot { issue.attachments.any { attachment -> attachment.name == getDeobfName(it.first) } }
+        minecraftCrashesWithDeobf.forEach {
+            issue.addAttachment(File(getDeobfName(it.first)))
+        }
+    }
+
+    private fun getDeobfName(name: String): String = "${name.substringAfterLast(".")}-deobfuscated.log"
```

Comment:
Crash reports typically have file extension `.txt`, not `.log`.

Reply:
Oh also shouldn't it be `substringBeforeLast`, since you're stripping away the extension? Otherwise this would always be `txt-deobfuscated.log`, wouldn't it?

Reply:
Java crashes (`hs_err_pid`) are `.log`, Minecraft crashes are all without exception `.txt`

Reply:
It would be nice to copy these deobfuticated crashed to the description for #281, but probably out of scope for this PR.